.gitignore

```diff
@@ -1,5 +1,6 @@
 /release/
 /drone-s3-cache*
+/.idea
```

Comment:
IDE ignores should not be part of the gitignore, that should be handled by each developer in a global gitignore configuration, otherwise we end up with trillions of different ignores.

Reply:
TIL. Sorry I didn't realize there could be a global .gitignore. A trillion seems a little far fetched since there are only a handful of popular ones that a majority of developers use, and some don't even store files. 😉 but I get your point.